seed/mappings/mapper.py

```diff
@@ -1,7 +1,6 @@
 """
 :copyright: (c) 2014 Building Energy Inc
```

Comment:
Sorry, I got thrown off because the automated testing says "This branch has no conflicts with the base branch." I assumed the base is the current develop branch, but that must not be the case.